tests/model_forms/test_modelchoicefield.py

```diff
@@ -260,6 +260,28 @@ class CustomModelChoiceField(forms.ModelChoiceField):
         self.assertIsInstance(field.choices, CustomModelChoiceIterator)
 
     def test_choice_iterator_passes_model_to_widget(self):
+        class CustomCheckboxSelectMultiple(CheckboxSelectMultiple):
+            def create_option(self, name, value, label, selected, index, subindex=None, attrs=None):
```

Comment:
Maybe `**kwargs` instead of `subindex=None, attrs=None`.

Reply:
Why? This is following the existing style of test `test_custom_choice_iterator_passes_model_to_widget` and the one in `docs/ref/forms/fields.txt`. I slightly prefer the explicit keywords, but not a big deal either way.

Reply:
Some tests are really old and it's not always the best to follow their style.
> Why?

To avoid mistakes, e.g. in the most recent push you fixed the next line:
```python
option = super().create_option(name, value, label, selected, index, subindex, attrs)
```
that wouldn't be necessary if we used  `kwargs`. But OK, it's fine.